Lib/cu2qu/__init__.py

```diff
@@ -67,7 +67,7 @@ def dot(v1, v2):
 
 def lerp(a, b, t):
     """Linearly interpolate between scalars a and b at time t."""
-    return a * (1 - t) + b * t
+    return a + (b - a) * t
```

Comment:
How do you mean? Spelling it out everywhere? We call lerp a lot in the bezier_at functions so I don't think that would be worth the messy code.

Anyways, I think my estimate of 5-10% was a bit much. The benchmark results aren't all that consistent. But it seems generally to be a bit faster.